### ISSUE TITLE
Confusing labeling in Custom Shells

### ISSUE THREAD

**Issue by swelljoe** [virtualmin/virtualmin-gpl#293]
The types of accounts in Custom Shells is confusing.

They are:  Admin? Mailbox? Reseller? Default?

Admin: Can easily be confused for "root" level users. I assume it is for domain owner users. It should be more clear that that's the case (maybe "Domain Admin").

Mailbox: it is not apparent that this would apply to users that also have FTP/SSH access.

Reseller: This one is fine.

Default: WTF is default? Default what?

Also, the ability to set the shell for `website FTP access users` is either impossible or named in such a way as to be completely opaque to users.

**Comment by swelljoe** [comment]
Actually, I'm misreading the table, I guess. Now that I think of it, I was confused by the years ago, too. Custom Shells is just a confusing UI all around.

**Comment by swelljoe** [comment]
So...I guess the question is how can this data be presented differently so that it makes sense to people who don't understand the technical details of how Virtualmin thinks about users?

**Comment by iliarostovtsev** [comment]
I agree that it may be confusing at first as you look at it but as you try it, it gets clearer on how it works and what it does.

I think the easiest work around for us would be is to add an ability for master administrator to have **`/bin/bash`** shell available on the dropdown when creating a new virtual server user.

> Default: WTF is default? Default what?

Joe, default means what shell on the select will be selected/set as default under *Create User / Other user permissions* or when creating a user on CLI.

I agree on that _Admin_ and _Mailbox_ labels must be improved. However, it's all described on the help link on the top left of the table.

> Custom Shells is just a confusing UI all around.

Well, for the start, I think we need to make sure that users wouldn't need using this page at all, by providing better defaults. Eventually, reworking this section seems like a lot of work, and this is not an easy concept to cover in the end and I am not sure if that would be possible to make it more simple in terms of saving usability and improving UX.

**Comment by swelljoe** [comment]
Yeah, I see how it works now (at least the "Default" bit). My brain just can never make this presentation make sense. I've been confused by it every couple of years from the very beginning. It's not comprehensible, though, for a new user.

I ask again: How can we present this data so it makes more sense? Certainly fixing the labels is an improvement. But, this grid of shells and user types is hella confusing.

**Comment by swelljoe** [comment]
Still doesn't make sense.

**Comment by iliaross** [comment]
Alright, should be fixed this time!
